src/OpenTelemetry.Instrumentation.AWSLambda/Implementation/AWSMessagingUtils.cs

```diff
@@ -145,7 +144,7 @@ internal static PropagationContext ExtractParentContext(SNSEvent.SNSMessage? mes
         {
             try
             {
-                snsMessage = JsonConvert.DeserializeObject<SNSEvent.SNSMessage>(body);
+                snsMessage = JsonSerializer.Deserialize<SNSEvent.SNSMessage>(body);
```

Comment:
You need `System.Text.Json` namespace.

Reply:
Ah, OK this does not even compile. Please see my above comment. I'm marking this as "Changes requested"